packages/widgets/src/downloads/component.tsx

```diff
@@ -726,15 +730,22 @@ const ItemInfoModal = ({ items, currentIndex, opened, onClose }: ItemInfoModalPr
           <NormalizedLine itemKey="index" values={item.index} />
           <NormalizedLine itemKey="type" values={item.type} />
           <NormalizedLine itemKey="state" values={t(item.state)} />
-          <NormalizedLine
-            itemKey="upSpeed"
-            values={item.upSpeed === undefined ? undefined : humanFileSize(item.upSpeed, "/s")}
-          />
+          {item.type === "torrent" && (
```

Comment:
I think this should probably instead be `!== "miscellaneous"`, because otherwise other integrations will also no longer show this (same for sent & ratio)